tsconfig.json

```diff
@@ -15,19 +15,11 @@
 		"isolatedModules": false,
 		"jsx": "react",
 		"module": "commonjs",
-		"plugins": [
```

Comment:
Any reason to remove this for this PR?

Reply:
Didn’t remove it entirely, was just a duplicate...